core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -106,6 +107,8 @@
             "META-INF/jandex.idx",
             "LICENSE")));
 
+    private static final String UNSET_VALUE = "<<unset>>";
```

Comment:
Let's make the constant public in `ApplicationInfoBuildItem` instead?

I think we already needed that somewhere else, it's definitely something that should be public if it needs to be consumed from somewhere else.